pkg/insights/insightsclient/insightsclient.go

```diff
@@ -304,34 +355,33 @@ func (c Client) RecvReport(ctx context.Context, endpoint string) (*io.ReadCloser
 	// dynamically set the proxy environment
 	c.client.Transport = clientTransport(c.authorizer)
 
-	klog.V(4).Infof("Retrieving report from %s", req.URL.String())
+	klog.V(4).Infof("Retrieving from %s", req.URL.String())
 	resp, err := c.client.Do(req)
-
 	if err != nil {
-		klog.Errorf("Unable to retrieve latest report for %s: %v", cv.Spec.ClusterID, err)
-		counterRequestRecvReport.WithLabelValues(c.metricsName, "0").Inc()
 		return nil, err
 	}
+	return resp, nil
+}
 
-	counterRequestRecvReport.WithLabelValues(c.metricsName, strconv.Itoa(resp.StatusCode)).Inc()
+func validateResponse(resp *http.Response) error {
 	requestID := resp.Header.Get("x-rh-insights-request-id")
```

Comment:
Maybe put this into a constant? Doesn't really matter. I don't see any conceptual issues, so I'm really just looking at these silly small things, sorry.

Reply:
No problem at all (some feedback is better than none :)). Thanks. I think our code base could generally use more constants ...